README.md

```diff
@@ -1,175 +1,404 @@
+
+Postgres Pro provides support along the full software development lifecycle—from code generation through testing, deployment, and production maintenance and tuning.
+
+*DEMO VIDEO PLACEHOLDER*
+
+## Contents
+
+- [Features](#features)
+- [Quick Start](#quick-start)
+- [Usage Examples](#usage-examples)
+- [MCP Server API](#mcp-server-api)
+- [Related Projects](#related-projects)
+- [Frequently Asked Questions](#frequently-asked-questions)
+- [Technical Notes](#technical-notes)
+- [Postgres Pro Development](#postgres-pro-development)
 
 ## Features
 
-postgres-mcp provides several powerful tools for database optimization:
+Postgres Pro offers a set of tools to help you query, analyze, and optimize your Postgres database.
+It provides:
+
+- **Schema Information**.
+  Help your AI Agent generate SQL reliably and successfully with detailed schema information of your database objects—including tables, views, sequences, stored procedures, triggers.
+
+- **Protected SQL Execution**.
+  Work fast or safe, as needed:
+  - *Unrestricted Mode:* Provide full read/write access in development environments. Let your AI agent modify data, change the schema, drop tables, whatever you need it to do.
+  - *Restricted Mode:* Limit access in production environments with checks to ensure read-only operations and limits on resource consumption. These restrictions may become more elaborate in the future.
+
+- **Index Tuning**.
+  Ensure your SQL queries run efficiently and return quickly.
+  Find tuning targets, validate AI-generated suggestions, or generate candidates using classical index optimization algorithms.
+  Simulate how Postgres will after adding indexes using the explain plans together with [hypothetical indexes](https://hypopg.readthedocs.io/).
+
+- **Database Health**.
+  Check buffer cache hit rates, identify unused/duplicate indexes, monitor vacuum health, and more.
 
-- **Database Health Analysis**: Check buffer cache hit rates, identify unused/duplicate indexes, monitor vacuum health, and more
-- **Query Performance Analysis**: Find your slowest SQL queries and get optimization suggestions
-- **Index Recommendations**: Analyze frequently executed queries and get optimal index suggestions
-- **Schema Exploration**: Explore database tables, columns, and relationships
 
 ## Quick Start
 
-#### Using Docker
+### Prerequisites
+
+Before getting started, ensure you have:
+1. Access credentials for your database.
+2. Python 3.12 or higher *or* Docker.
+3. The `pg_statements` and `hypopg` extensions loaded on your database (for full functionality).
+
+#### Access Credentials
+ You can confirm your access credentials are valid by using `psql` or a GUI tool such as [pgAdmin](https://www.pgadmin.org/).
+
+
+#### Python  or Docker
+
+The choice to use Docker or Python is yours.
+We generally recommend using whichever is most familiar to you.
+
+#### Postgres Extensions
+
+Postgres Pro uses the `pg_statements` extension to analyze query execution statistics.
```

Comment:
`pg_statements` or `pg_stat_statements`?
Maybe break this section into 2 sections?

##### High value (or something to talk about the value of enabling pg_stat_statements`
##### Highest value (but requires some effort)